os/arch/arm/src/sidk_s5jt200/src/s5jt200_tash.c

```diff
@@ -271,6 +271,7 @@ int board_app_initialize(void)
 	struct mtd_dev_s *mtd;
 #endif /* CONFIG_RAMMTD */
 
+#ifndef CONFIG_DISABLE_MOUNTPOINT
```

Comment:
In artik053, you added a dependancy at Kconfig. Do you want to add a conditional in source code for s5jt200?